### DISCUSSION TITLE
appConfig.auth issue

### DISCUSSION THREAD

**Discussion by fosteraa** [Lissy93/dashy#327] [Help and Support] [answered]
I've been testing the auth configuration, with the basic premise that guests would have read-only access, and a named user would have permissions to change config and add sections/links. I have configured 

'Authentication - Enable Guest Mode' enabled
and
a generic admin user and password created in 'Users'

Logging in and out with the admin user, and logging in as guest, work as expected. However, the issue is that when not logged in (guest mode) I can still edit the config and save it to disk.

**Comment by lissy93** [comment, marked as answer]
Update: Have looked into this, if you are only logged in as a guest, then when you save config it is only applied locally (no changes are made to the conf.yml file). This is the expected behavior.

Edit: Ahh, you mean the new editor? Have pushed an update which should fix this :)
